docs/topics/con_nodejs-runtime-design-details.adoc

```diff
@@ -1,5 +1,5 @@
-[id='nodejs-runtime-details_{context}']
-= {Node} runtime details
+[id='nodejs-runtime-design-details_{context}']
+= {Node} runtime design details
```

Comment:
I'm not sure what's this supposed to mean. We don't really provide details around Node.js design.

Reply:
Since we're talking about what Node.js is, how it works and what are its advantages, it seemed to me this would be a more specific heading. Do you have any other idea? I'm stumped.

Reply:
Actually we don't really dive into much detail either. It's an overview, and not really from a design perspective (where I understand the word "design" to mean "software design"). Something like "What is Node.js" or "Node.js overview" sounds better to me.

(I also think I've seen the same change for Spring Boot. Not sure about other runtimes.)

Reply:
Okay, I suppose "What is Node.js" would indeed work, thank you. I'll file a follow-up PR.